doc/htmldoc/conf.py

```diff
@@ -330,21 +286,18 @@ def setup(app):
 
 
 def copy_example_file(src):
-    copyfile(src, doc_build_dir / "static/img" / src.parts[-1])
-
-
-def copy_acknowledgments_file(src):
-    copyfile(src, doc_build_dir / src.parts[-1])
-
+    copyfile(os.path.join(pynest_dir, src), Path("examples") / Path(src).parts[-1])
 
-# -- Copy Acknowledgments file ----------------------------
-copy_acknowledgments_file(source_dir / "ACKNOWLEDGMENTS.md")
```

Comment:
Don't we need to copy this file anymore?

Reply:
@jessica-mitchell explained that the achknowledgements are now automatically transcluded in some other way?

Reply:
The acknowledgments in master is a link in index.html to the github page

Reply:
Alright. Thanks for the clarification.